queries/java/highlights.scm

```diff
@@ -103,7 +105,7 @@
 ((identifier) @constant
   (#match? @constant "^_*[A-Z][A-Z\d_]+"))
 
-
+(this) @constant.builtin
```

Comment:
I think that we went for `variable.builtin` in other languages.

Reply:
`variable.builtin` never happened. Other languages, use `constant.builtin`. Could change this for all languages.

Reply:
Nope, keep it this way if it is consistent